File: docs/datatools/codesnippet/CSharp/how-to--execute-a-stored-procedure-that-returns-a-single-value_2.cs
            SqlConnection sqlConnection1 = new SqlConnection("Your Connection String");
            SqlCommand cmd = new SqlCommand();
            Object returnValue;
            
            cmd.CommandText = "StoredProcedureName";
            cmd.CommandType = CommandType. StoredProcedure;
            cmd.Connection = sqlConnection1;

            sqlConnection1.Open();

            returnValue = cmd.ExecuteScalar();

            sqlConnection1.Close();

File: docs/data-tools/codesnippet/VisualBasic/how-to-execute-a-stored-procedure-that-returns-a-single-value_2.vb
        Dim sqlConnection1 As New SqlConnection("Your Connection String")
        Dim cmd As New SqlCommand
        Dim returnValue As Object

        cmd.CommandText = "StoredProcedureName"
        cmd.CommandType = CommandType.StoredProcedure
        cmd.Connection = sqlConnection1

        sqlConnection1.Open()

        returnValue = cmd.ExecuteScalar()

        sqlConnection1.Close()